### ISSUE TITLE
Input for mutation type can only bind data in the last field

### ISSUE THREAD

**Issue by thangchung** [pekkah/tanka-graphql#93]
I'm not sure if it is a bug or not, but when I created the `input` type with multiple fields like

```gql
input CreateProductInput {
  name: String!
  price: Float!
  imageUrl: String
  desc: String
}
``` 

with `mutation` type

```gql
type Mutation {
  createProduct(message: CreateProductInput!): Product!
}
```

```gql
schema {
  query: Query
  mutation: Mutation
}
```

on the server side, I have created `CoolStoreResolvers` class

```csharp
public class CoolStoreResolvers : ResolverMap
{
    public CoolStoreResolvers(ICoolStoreResolverService resolverService)
    {
        
        this["Mutation"] = new FieldResolverMap
        {
            {"createProduct",  resolverService.CreateProductAsync}
        };
    }
}
```

and `CoolStoreResolverService` as

```csharp
public class CoolStoreResolverService : ICoolStoreResolverService
{
    public async Task<IResolveResult> CreateProductAsync(ResolverContext context)
    {
        var input = context.GetArgument<CreateProductRequest>("message");
		
		// do something		

        return await Task.FromResult(As(product));
    }
}
```

I tested with `graphiql` with the test data like

```gql
mutation createProduct($message: CreateProductInput!) {
  createProduct(message: $message) {
    name
	price
	desc
  }
}
```

Input variables as below

```json
{
  "message": {
    "name": "product 1",
    "price": 100,
    "desc": "this is product 1"
  }
}
``` 

In the code above, I always got only `desc` field binding data with the string `this is product 1`, and the rest of fields such as `name`, `price` and `imageUrl` were null.

I have seen the [`samples` folder](https://github.com/pekkah/tanka-graphql/blob/master/samples/graphql.samples.chat.data/idl/schema.graphql#L14) from this project only have one field for `input` type as

```gql
input InputMessage {
    content: String!
}
``` 

Is it because this library could not bind multiple field data for `input` type?

I used 

```csproj
<PackageReference Include="tanka.graphql" Version="0.3.1" />
<PackageReference Include="tanka.graphql.server" Version="0.3.1" />
```

**Comment by pekkah** [comment]
It should work. I'll check it out later.

**Comment by pekkah** [comment]
It seems to be working on at least on the executor side https://github.com/pekkah/tanka-graphql/blob/2f2119aa1f7300072ed0e586b89db42507ffcf4e/tests/graphql.tests/ExecutorFacts.cs#L173.

I'll check if the SignalR is somehow messing up the serialization of the input variables.

**Comment by pekkah** [comment]
Found the problem and fixed it. Fix will be in latest beta as soon as the builds finish and I can merge the PR.

**Comment by pekkah** [comment]
Fixed in 0.3.2

**Comment by thangchung** [comment]
@pekkah Thank you very much. It works. I can create the `input`s at https://github.com/vietnam-devs/coolstore-microservices/blob/master/src/graphql/coolstore.graphql#L10
I made a ref to https://github.com/vietnam-devs/coolstore-microservices/issues/41 for remembering
